### DISCUSSION TITLE
Super basic help request?

### DISCUSSION THREAD

**Discussion by pistachoo** [CANDY-HOUSE/SesameSDK_Android_with_DemoApp#42] [Q&A]
I can't find anywhere to find basic level tech support with the Sesame lock. I've searched everywhere for hours.

How do I reset the device to pair with my phone? The app says "To reset the registered device, please press the reset button with a pin until the LED light turns off." However, it never turns off.

Help?

**Comment by pistachoo** [comment]
https://imgur.com/a/XalFek5

**Reply by Jerming** [reply]
https://apps.apple.com/app/sesame-by-candy-house/id1038881712
https://drive.google.com/file/d/1fvgpEIAWFp9il3Ci7hEO820iZMStCe20/view?usp=sharing

Please use the app, thanks.

**Comment by pistachoo** [comment]
I was using the app from the Google Play store.

The APK you sent me doesn't install.

**Reply by Jerming** [reply]
Hi @pistachoo, thanks for replying.
The Sesame you have was developed in 2015 and is not supported by the current app on the Google Play Store.
We encourage you to use an iPhone/iPad/Mac with an Apple Silicon M1/2/3 chip to use the app from the App Store, or find a Pixel 6/5/4/3/2/1 to install the app from a Google Drive link.

Or, if you are willing to wait a bit longer, we will soon be launching Sesame5 for the USA in January 2025, by which time you will be able to enjoy the latest software.

**Comment by pistachoo** [comment]
Oh, so you're saying my Sesame is too old, AND my phone is too new?

So I am doubly hooped.

Ugh.

**Reply by Jerming** [reply]
I am sorry. Let me send you free one when Sesame 5 is available in America.

**Comment by pistachoo** [comment]
A free lock? :O

That would be very generous of you.

In the meantime, I did get it to work with an old phone, but I cannot add
myself as a friend, so it really is completely obsolete and incompatible.
Too bad. I guess there is no way to make it work with WiFi things either
(not the official ones, since it's so old, but like the Arduino or RPi
controller types) as workarounds... I'll just have to wait.

I keep locking myself out when I walk the dog and forget my keys, haha!

:)

Geneviève


--

'Sometimes I think the surest sign that intelligent life exists elsewhere
in the universe is that none of it has tried to contact us.'

~ Calvin ~
